app/helpers/pages_helper.rb

```diff
@@ -182,6 +182,19 @@ def stylesheet_pack_tag_for_plugin(plugin)
     stylesheet_pack_tag pack_name if pack_exists?(pack_name)
   end
 
+  def page_object(page)
+    exceptions = %i[content javascript liquid_layout_id compiled_html messages]
+    base = page.as_json(except: exceptions)
```

Comment:
I would explicitly select the fields that we need instead of using `except`. The current way of doing creates an implicit coupling between page model schema and the page_object. If we add a field or remove a field from the table, this will have an undesired consequence here.